common/lib/actions/add_comment.dart

```diff
@@ -7,8 +7,9 @@ part 'add_comment.g.dart';
 class AddCommentRequest {
   final int pollId;
   final String comment;
-
-  const AddCommentRequest({required this.pollId, required this.comment});
+  final int? parentId; //null if top-level comment
```

Comment:
Can you rewrite this comment as a Dartdoc comment?

See [here](https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph) for an example.

Reply:
Oops, that's a mistake. I later changed `parentId` to be zero for top-level comments. if this is bad practice we can change to null, so I'll hold off on the Dartdoc comment for now.